state/accountsDB.go

```diff
@@ -1206,7 +1217,11 @@ func (adb *AccountsDB) processSnapshotCompletion(
 ) {
 	adb.finishSnapshotOperation(rootHash, stats, missingNodesCh, message, trieStorageManager)
 
-	defer adb.isSnapshotInProgress.Reset()
+	defer func() {
+		adb.isSnapshotInProgress.Reset()
+		adb.appStatusHandler.SetStringValue(common.MetricTrieSnapshotIsProgress, strconv.FormatBool(adb.isSnapshotInProgress.IsSet()))
```

Comment:
[question]

Couldn't have we simply set it to false here?

Reply:
i would keep it as it is to make the connection with snapshotInProgress var clear